shared/lib-metrics/src/main/java/gov/va/vro/metricslogging/MetricLoggerService.java

```diff
@@ -19,6 +20,21 @@ public class MetricLoggerService implements IMetricLoggerService {
 
   private final MetricsApi metricsApi;
 
+  @Value("${vro.env}")
+  public final String env;
+
+  @Value("${vro.it-portfolio:benefits-delivery}")
+  public final String itPortfolio;
+
+  @Value("${vro.team:va-abd-rrd}")
+  public final String team;
+
+  @Value("${vro.app.name}")
```

Comment:
Non-blocking. Can we make this consistent with the name of the variable so that in my properties, i know this will correspond directly to the `service` tag?

Reply:
Good call out, I'll update this.